src/js/components/PeaksList/Peak.tsx

```diff
@@ -20,10 +20,14 @@ const Peak = ({ peak, redirectTo, className }: PeakProps) => {
           <div className={styles.peakName}>{peak.name}</div>
           <div className={styles.peakHeight}>{peak.height} m n.p.m.</div>
         </div>
-        <div className={styles.rating}>
-          <AiFillStar className={styles.star} />{' '}
-          <p className={styles.ratingText}>4.9/5</p>
-        </div>
+      </div>
+      <div className={styles.reachedInfo}>
+        {peak.completed ? (
+          <TiTick className={styles.reachedIcon} />
+        ) : (
+          <TiTimes className={styles.reachedIcon} />
+        )}
+        <p className={styles.reachedText}>Reached</p>
```

Comment:
Może zmieniajmy tekst z Reached na Reach jeśli ktoś jeszcze nie zdobył szczytu i może podmienimy kolor na jakiś inny po zdobyciu? Co o tym sądzisz?

Reply:
W sumie możemy, na razie dałam tak, bo na mockach nie było to uwzględnione. Jaki kolor proponujesz? Zielony?

Reply:
Zielony wydaje mi się ok, mamy go trochę w aplikacji